src/main/java/com/google/sps/servlets/MentorSignupServlet.java

```diff
@@ -33,8 +36,40 @@
 public class MentorSignupServlet extends HttpServlet {
 
   /**
-   * Receives information about a new mentor and stores it in the database.
+   * Gets mentor experience tags from database and returns as JSON.
    */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Set up list to store subject tags.
+    List<SubjectTag> subjectTags = new ArrayList<SubjectTag>();
+    
+    // Set up query to retrieve all subject tags.
+    String query = "SELECT * FROM SubjectTag";
+
+    try {
+      // Establish connection to MySQL database.
+      Connection connection = DriverManager.getConnection(Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+
+      // Create the MySQL prepared statement, execute it, and store the result.
+      PreparedStatement preparedStatement = connection.prepareStatement(query);
+      ResultSet queryResult = preparedStatement.executeQuery();
+
+      // Store queryResult in map and close connection to database.
```

Comment:
nit: I think queryResult is being stored in a list, and not a map?

Reply:
Yup! Done.